### DISCUSSION TITLE
Does the manager check if the temp drive has enough space for the entire plotting cycle before starting a plot?

### DISCUSSION THREAD

**Discussion by junhua** [swar/Swar-Chia-Plot-Manager#1197] [Q&A]
Does the manager check if the temp drive has enough space for the entire plotting cycle before starting a plot?

**Comment by Out-Grow** [comment]
No it does not. You will need to calculate the available space on your temp drive for a max of 256G temp space per plot. If you have a 1TB temp drive You can process 4-5 plots on it max if staggered correctly.

**Reply by junhua** [reply]
It'd be a good feature to have; Also I realised that the calculation of destination drive is not accurate as it doesn't take into consideration of existing plotting jobs.
